NimbleMedium/Sources/Presentation/Modules/ArticleDetail/ArticleDetailView.swift

```diff
@@ -153,6 +155,20 @@ struct ArticleDetailView: View {
         }
     }
 
+    func unAuthorButtons(uiModel: UIModel) -> some View {
```

Comment:
maybe `interactionButtons` sounds better 🙏

```suggestion
    func interactionButtons(uiModel: UIModel) -> some View {
```